src/figuresClasses.ts

```diff
@@ -1,11 +1,80 @@
-export interface Figure {}
+export type Shape = 'triangle' | 'circle' | 'rectangle';
+export type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
```

Comment:
```suggestion
  shape: Shape = 'triangle';
```

The same in other places